iOS/Examples/StitchLocalMongoDBServiceExample/StitchLocalMongoDBServiceExample/AppDelegate.swift

```diff
@@ -1,5 +1,5 @@
 import UIKit
-import StitchCore_iOS
+import StitchCore
```

Comment:
[minor] I think from now on, we should be importing the uber module in examples (`StitchSDK`), people will probably look at examples and copypaste. Plus it'll be good a test for us to make sure anything that people will need to build apps is exposed in the module

Reply:
StitchLocalMongoDBService is specifically not part of the uber module though, so that seems a bit misleading here.

Reply:
Yes, so you can have both `import StitchSDK` and `import StitchLocalMongoDBService`. `StitchCore` shouldn't need to be imported by people even when they're using the local mongo service.